src/services/oauthService.ts

```diff
@@ -40,43 +42,14 @@ export class OAuthService {
         return text;
     }
 
-    public async getAuthServer(apiId: string): Promise<AuthorizationServer> {
+    public async getOpenIdAuthServers(apiId: string): Promise<AuthorizationServer[]> {
         try {
-            let authServer: AuthorizationServer;
-
-            authServer = await this.getOauth2ServerByApiId(apiId);
-
-            if (!authServer) {
-                authServer = await this.getOpenidconnectServerByApiId(apiId);
-            }
-
-            return authServer ?? undefined;
+            const authServers = await this.apiClient.get<Page<AuthorizationServerForClient>>(`apis/${apiId}/authServers/openidconnect`, [await this.apiClient.getPortalHeader("getAuthorizationServer"), Utils.getIsUserResourceHeader()]);
```

Comment:
you are only fetching openidconnect servers, arent we supposed to fetch oauth2 servers as well :) also, isn't there any use case where we fetch only a single server?

Reply:
hmm, I see, but why only Oauth2 is coming together with API? Why dont we have openid with it? can you take a look at why it is like this? Also I am not sure if it is the case but, in case oauth2 settings are empty, you continue as it is instead of fetching the oauth2 server.

![image](https://user-images.githubusercontent.com/26141403/194277281-977dc05a-a456-4c85-ae8b-cf6498bd7e21.png)

I think between those, you can fetch oauth2 if it is not returned with api to make sure.

Reply:
As discussed, I made the change to also make the call for OAuth2 servers.